hack/get_entrypoint.sh

```diff
@@ -46,6 +46,17 @@ for VERSION_DIR in $(find mariadb-docker/ -maxdepth 1 -type d -regex 'mariadb-do
   fi
 done
 
+ENTRYPOINT_PATH=mariadb-docker/main-ubi
```

Comment:
@grooverdan could you provide some context about this folder? Why getting the entrypoint from `main-ubi` and not from `main`? Are these development versions i.e. 11.8 currently?

Leaving a small comment would be appreciated.

Reply:
was mainly because the version was easier to parse out of the Dockerfile in the ubi dir vs the ubuntu version in its dockerfile.

Reply:
Fair enough, thanks for the context.